internal/git.go

```diff
@@ -69,16 +70,30 @@ func GetGitConfigOrg(path string) string {
 		return ""
 	}
 
-	splitOrg := strings.SplitN(url.Path, "/", 2)
+	var (
+		splitOrg []string
+		org      string
+	)
+
+	// some ssh and https url paths start with a /, some with a :
+	// therefore, split logic is different
+	if style, _ := regexp.MatchString(`^\/`, url.Path); style {
```

Comment:
can we simplify this a bit by doing a `strings.TrimPrefix(url.Path, "/")` and running the result of that through `SplitN` with `2`? we can avoid regex and some if/else statements